### PR TITLE
Added unit test for POST book feature.

### PR DESCRIPTION
**Pull request by theTytoAlba**
Added unit test for POST book feature.

### CODE REVIEW

File: Book API Homework/bookapi/bookapi/settings.py [hunk 1]
```diff
@@ -107,6 +107,8 @@
 
 LANGUAGE_CODE = 'en-us'
 
+DEFAULT_CHARSET='utf-8'
```
Comment (akinarpacioglu) [inline Book API Homework/bookapi/bookapi/settings.py]: What is this for?

**Comment by akinarpacioglu** [comment]
Can you update the readme file with the unit testing steps please?

File: Book API Homework/bookapi/bookapi/settings.py (hunk 1 above)
Comment (theTytoAlba) [reply Book API Homework/bookapi/bookapi/settings.py]: Removed the unnecessary change. Thanks for noticing! :)

**Review by akinarpacioglu — APPROVED** [review]
Thanks!

**Comment by sevgicansalih** [comment]
It is quite nice thank you very much for your efforts. I'll merge the code

**Pull request merged**